src/HPCCache/HPCCache.Test/HPCCache.Test.csproj

```diff
@@ -17,7 +17,7 @@
   <ItemGroup>
     <PackageReference Include="Microsoft.Azure.Management.Authorization" Version="2.11.0-preview" />
     <PackageReference Include="Microsoft.Azure.Management.Storage" Version="14.5.0" />
-    <PackageReference Include="Microsoft.Azure.Management.Network" Version="20.0.1-preview" />
+    <PackageReference Include="Microsoft.Azure.Management.Network" Version="19.18.0-preview" />
```

Comment:
please confirm this version, I think officially released is 19.21.0?

Reply:
This 19.18.0-preview is what is the network-may branch and in master. I don't want to upgrade HpcCache.Test project due to the issue I emailed about earlier. So keeping it unaltered at 19.18.0-preview.